addons/udes_security/models/allowed_file_type.py

```diff
@@ -2,6 +2,27 @@
 from odoo.exceptions import UserError
 
 
+class MimeType(models.Model):
+    """Store mime types related to allowed file types."""
+
+    _name = "udes.mimetypes"
+    _description = "Mimetypes related to allowed file types."
+
+    file_type_ids = fields.Many2many(
```

Comment:
Is needed this many2many field?

Reply:
We don't actually use it, so it can be removed (tbh I wasn't sure whether Odoo required the declaration in both models or not).

Reply:
No its not required, i think only in one the one which you are gonna use should be fine.

Reply:
Removed in 4e274cbdd0c97c8af2754edfbb750d9574e4b087